sequential/quadtree.cpp

```diff
@@ -0,0 +1,248 @@
+#include<bits/stdc++.h>
+
+using namespace std;
+
+// Create the definition of a point for the QuadTree
+struct Point {
+    // Each point is defined as (x,y)
+    int x;
+    int y;
+
+    Point(int xVal = 0, int yVal = 0){
+        x = xVal;
+        y = yVal;
+    }
+};
+
+// The Nodes of a QuadTree
+struct Node {
+    Point NodePos;
+    int data;
+    
+    // Node contains coordinates and data
+    Node(Point p, int d){
+        NodePos = p;
+        data    = d;
+    }
+};
+
+// QUADTREE
+class QuadTree {
```

Comment:
Instead of using 3 different structs, can't we just have 2 structs - Node and QuadTree? Within Node we can store it's x and y coordinates as well. QuadTree can also store coordinates of Top Right and Bottom Left corner.

Reply:
I can try and update it.